### PR TITLE
add more desc about replicatuin-factor

### PR DESCRIPTION
**Pull request by glkappe**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [ ] master (the latest development version)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.4 (TiDB 7.4 versions)
- [ ] v7.3 (TiDB 7.3 versions)
- [ ] v7.2 (TiDB 7.2 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v7.0 (TiDB 7.0 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by CharlesCheung96 — APPROVED** [review]

File: ticdc/ticdc-sink-to-kafka.md [hunk 1]
```diff
@@ -56,7 +56,7 @@ URI 中可配置的的参数如下：
 | `kafka-client-id`    | 指定同步任务的 Kafka 客户端的 ID（可选，默认值为 `TiCDC_sarama_producer_同步任务的 ID`）。 |
 | `partition-num`      | 下游 Kafka partition 数量（可选，不能大于实际 partition 数量，否则创建同步任务会失败，默认值 `3`）。|
 | `max-message-bytes`  | 每次向 Kafka broker 发送消息的最大数据量（可选，默认值 `10MB`）。从 v5.0.6 和 v4.0.6 开始，默认值分别从 64MB 和 256MB 调整至 10 MB。|
-| `replication-factor` | Kafka 消息保存副本数（可选，默认值 `1`）。                       |
+| `replication-factor` | Kafka 消息保存副本数（可选，默认值 `1`），需要大于等于 kafka 中 min.insync.replicas 值。                       |
```
Comment (ran-huang) [inline ticdc/ticdc-sink-to-kafka.md:59]:
```suggestion
| `replication-factor` | Kafka 消息保存副本数（可选，默认值 `1`），需要大于等于 Kafka 中 `min.insync.replicas` 的值。 |
```

**Review by ran-huang — APPROVED** [review]

**Comment by qiancai** [comment]
In https://docs.pingcap.com/tidb/dev/hybrid-deployment-topology#topology-templates, there is a link to https://github.com/pingcap/docs-cn/blob/master/config-templates/simple-multi-instance.yaml. Before removing this yaml, this link should be updated too.

File: ticdc/ticdc-sink-to-kafka.md [hunk 2]
```diff
@@ -56,7 +56,7 @@ URI 中可配置的的参数如下：
 | `kafka-client-id`    | 指定同步任务的 Kafka 客户端的 ID（可选，默认值为 `TiCDC_sarama_producer_同步任务的 ID`）。 |
 | `partition-num`      | 下游 Kafka partition 数量（可选，不能大于实际 partition 数量，否则创建同步任务会失败，默认值 `3`）。|
 | `max-message-bytes`  | 每次向 Kafka broker 发送消息的最大数据量（可选，默认值 `10MB`）。从 v5.0.6 和 v4.0.6 开始，默认值分别从 64MB 和 256MB 调整至 10 MB。|
-| `replication-factor` | Kafka 消息保存副本数（可选，默认值 `1`）。                       |
+| `replication-factor` | Kafka 消息保存副本数（可选，默认值 `1`），需要大于等于 Kafka 中 `min.insync.replicas` 的值。 |
```
Comment (qiancai) [inline ticdc/ticdc-sink-to-kafka.md:59]:
```suggestion
| `replication-factor` | Kafka 消息保存副本数（可选，默认值 `1`），需要大于等于 Kafka 中 [`min.insync.replicas`](https://kafka.apache.org/33/documentation.html#brokerconfigs_min.insync.replicas) 配置项进行配置） 的值。 |
```

**Review by qiancai — APPROVED** [review]
Rest LGTM

**Pull request merged**